scripts/prepdocslib/textsplitter.py

```diff
@@ -105,3 +108,29 @@ def find_page(offset):
 
         if start + self.section_overlap < end:
             yield SplitPage(page_num=find_page(start), text=all_text[start:end])
+
+
+class SimpleTextSplitter(TextSplitter):
+    """
+    Class that splits pages into smaller chunks. This is required because embedding models may not be able to analyze an entire page at once
```

Comment:
Please adjust the docstrings to be distinct across these two classes. Also dont think the original docstring is 100% correct, as we also chunk to reduce the context sent to the LLM. Thats primary purpose in my mind. Perhaps you can mention both.